recipes/eemont/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "eemont" %}
+{% set version = "0.1.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ee2f693fe5b988fe686773717c9c833e0d62772549b50444edd70d72ecb77719
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - earthengine-api
```

Comment:
Looks like `requests` is also listed in `setup.py`?

Reply:
Sorry, my bad! I just added it.